.github/workflows/build.yml

```
@@ -26,8 +26,9 @@ jobs:
       with:
         path:
           ~/.m2
-        key: build-${{ env.cache-name }}
+        key: ${{ runner.os }}-${{ matrix.java_version }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-${{ matrix.java_version }}-maven-
     - name: Maven version
       run: mvn --version
     - name: Build with Maven
-      run: mvn clean install && mvn clean verify artifact:compare && rm -rf ~/.m2/io/github/cbarlin
\ No newline at end of file
+      run: rm -rf ~/.m2/io/github/cbarlin && mvn clean install && mvn clean verify artifact:compare && rm -rf ~/.m2/io/github/cbarlin
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)